docs/source/conf.py

```diff
@@ -0,0 +1,95 @@
+"""Configuration file for the Sphinx documentation builder.
+
+For the full list of built-in configuration values, see the documentation:
+https://www.sphinx-doc.org/en/master/usage/configuration.html
+"""
+
+from importlib.metadata import metadata
+
+project_metadata = metadata("dragonfly-mainframe")
+project: str = project_metadata["Name"]
+release: str = project_metadata["Version"]
+REPO_LINK: str = project_metadata["Project-URL"].replace("repository, ", "")
+copyright: str = "Let's build a ..."  # noqa: A001
+author: str = "Let's build a ... community"
```

Comment:
Change copyright to `Vipyr Security` and author to `Vipyr Security Developers` or similar.

Reply:
88072395ba91270915f8a610261e8b52ea3f1ec9